ui/app/components/app/index.scss

```diff
@@ -87,3 +87,9 @@
 @import 'multiple-notifications/index';
 
 @import 'signature-request/index';
+
+@import 'multiple-notifications/index';
```

Comment:
A number of styles seem to be re-imported here for some reason. Maybe a mistake during conflict resolution.

Reply:
What do you mean by "re-imported"?

Reply:
Nevermind, I get it. Fixed in 43e13a98a